tests/Unit/Helper/MiscHelperTest.php

```diff
@@ -33,4 +33,12 @@ public function getInputOutputForSplitSecondsMicroseconds()
             array('3.123456', array(3, 123456)),
         );
     }
+
+    public function testHexDump()
+    {
+        $res = MiscHelper::hexdump('FM', 'hmtlOutput' == false, 'uppercase' == false, 'return' == true);
```

Comment:
Awright, awright :-)

I'm going to commit a non-hack, cleaned-up call. Give me 10 mins.